src/ui/server/controllers/insurance/feedback/feedback-form/index.ts

```diff
@@ -95,9 +111,20 @@ const post = async (req: Request, res: Response) => {
         [OTHER_COMMENTS]: feedback[OTHER_COMMENTS],
       } as InsuranceFeedbackVariables;
 
+      // variables for adding feedback to the database
+      const databaseVariables = {
+        [SATISFACTION]: feedback[SATISFACTION] ?? '',
+        [IMPROVEMENT]: feedback[IMPROVEMENT],
+        [OTHER_COMMENTS]: feedback[OTHER_COMMENTS],
+        [TYPE]: 'Insurance',
```

Comment:
Can we move this into a constant - also "type" field name feels confusing, how about `service` `SERVICE`?

I think it'd be a good idea to add default `serviceName` or `product` as well with a value of "EXIP", for future data consumption outside of our DB. Not sure about the namings, what do you think?